cmd/root.go

```diff
@@ -135,11 +137,17 @@ var detectHelmCmd = &cobra.Command{
 	Short: "detect-helm",
 	Long:  `Detect Kubernetes apiVersions in a helm release (in cluster)`,
 	Run: func(cmd *cobra.Command, args []string) {
-		h := helm.NewHelm(helmVersion, namespace)
+		if helmStore != "secrets" && helmStore != "configmaps" && helmVersion == "2" {
+			fmt.Println("helm-store should be configmaps or secrets")
+			os.Exit(1)
+		}
+		if helmVersion == "3" && helmStore != "" {
+			fmt.Println("helm-store work only with helm v2")
+		}
+		h := helm.NewHelm(helmVersion, helmStore, namespace)
 		err := h.FindVersions()
 		if err != nil {
-			fmt.Println("Error running helm-detect:", err)
-			os.Exit(1)
```

Comment:
the `os.Exit(1)` line should not be removed.

Reply:
I have a case where the official helm chart have a "bug", and instead of just exiting with error, it print the error and continue.

```text
./pluto detect-helm --helm-version 2 --helm-store secrets
I0528 11:46:57.737333  109006 root.go:150] Error running helm-detect: error parsing release 'trickster'
   yaml: unmarshal errors:
  line 46: mapping key "name" already defined at line 39

NAME                                                            KIND                           VERSION                                REPLACEMENT                       REMOVED   DEPRECATED  
application-status-agent-prod/application-status-agent-prod     Ingress                        extensions/v1beta1                     networking.k8s.io/v1beta1         false     true        
cert-manager/cert-manager-webhook                               MutatingWebhookConfiguration   admissionregistration.k8s.io/v1beta1   admissionregistration.k8s.io/v1   false     true        
cluster-autoscaler/cluster-autoscaler-aws-cluster-autoscaler    Deployment                     extensions/v1beta1                     apps/v1                           true      true        
```
Do you have a suggestion about how to deal with this kind of error?

Reply:
Yes that's an issue with the chart like you mentioned. We can't workaround that easily since it's a yaml marshaling error. It might be possible, but I looked into it a while back and it seems easier/better to upgrade the chart to a version with the bugfix, or open a PR tot he chart to fix it. I don't think removing this line is the answer though, and we can dig into that in a different PR/Issue.